client/my-sites/domains/domain-management/settings/cards/domain-forwarding-card.tsx

```diff
@@ -78,7 +84,7 @@ export default function DomainForwardingCard( {
 		if ( isError ) {
 			dispatch(
 				errorNotice(
-					translate( 'An error occurred while fetching your domain redirects.' ),
+					translate( 'An error occurred while fetching your domain forwarding.' ),
```

Comment:
Would like to gain alignment on this: unifying that we won't be adding s to `forwarding`?
Context: My spell checker suggests that there's no such thing as `forwardings`